src/@types/styled-components.d.ts

```diff
@@ -20,6 +20,12 @@ declare module 'styled-components' {
         defaultLight: string;
         header: string;
         headerLight: string;
+        headerLight: string;
```

Comment:
This was already in, can be removed

Reply:
![Whoopsies...](https://media0.giphy.com/media/3ohzdYJK1wAdPWVk88/giphy.gif?cid=ecf05e47373426cd96a67d8564354c3c88265fb478680c8c&rid=giphy.gif)